.github/workflows/README.md

```diff
@@ -2,7 +2,7 @@
 
 This directory contains workflows to be used for Lessons using the {sandpaper}
 lesson infrastructure. Two of these workflows require R (`sandpaper-main.yaml`
-and `pr-receive.yaml`) and the rest are bots to handle pull request management.
+and `pr-recieve.yaml`) and the rest are bots to handle pull request management.
```

Comment:
Not sure where these changes `receive` -> `recieve` are coming from? The later is the correct spelling, and used in [sandpaper](https://github.com/carpentries/sandpaper/blob/main/inst/workflows/pr-receive.yaml)